python/main.py

```diff
@@ -0,0 +1,166 @@
+import os
```

Comment:
where did you document how to use this file ?

Reply:
What do you mean...? I explained how to run this script in the root ReadMe.md